v5/releases/index.rst

```diff
@@ -6,7 +6,6 @@ Kernel Releases
 ===============
 
 .. postlist:: 5
-   :sort:

```

Comment:
We will finally have 6 releases. An apparently `:sort:` makes it chronological from
oldest
...
newest

when we want the other way around

Reply:
will it autosort them correctly then?